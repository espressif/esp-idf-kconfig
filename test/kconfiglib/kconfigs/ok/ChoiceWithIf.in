mainmenu "Test if statement inside Choice with condition depending on choice"

    choice
        prompt "Test if statement inside Choice"
	    default FOO

        config FOO
            bool "foo"

        if FOO

            config BAR # BAR will NOT be part of the choice
                 bool "bar"

        endif

    endchoice
