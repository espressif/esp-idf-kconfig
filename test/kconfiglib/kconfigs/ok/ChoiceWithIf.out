#
# Automatically generated file. DO NOT EDIT.
# Espressif IoT Development Framework (ESP-IDF)  Project Configuration
#
CONFIG_FOO=y
# default:
# CONFIG_BAR is not set
